### ISSUE TITLE
Colon can be used instead of slash in var declarations

### ISSUE THREAD

**Issue by DamianX** [SpaceManiac/SpacemanDMM#36]
```
/obj
    var:last_movement_dir = 0
```

The above code seems to work correctly, for some reason.
At the moment, `dmm-tools check` reports it as an error and aborts:

```
code/modules/power/singularity/singularity.dm, line 40, column 5:
error: got ':', expected one of: '/', '[', contents, newline, '}', identifier
```

I found out that an arbitrary amount of `:.` is also happily accepted by DM.
```
/obj
    var:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.last_movement_dir = 0
```

This should probably be turned into a warning.

**Comment by SpaceManiac** [comment]
When I was testing this, I checked `/` (which works) and `.` (which doesn't), but forgot to test `:`. Parsing should be tweaked to accept a colon here. A construct like `:.:.:.` falls into the explicitly non-supported category of "dumb and meaningless code that DM accepts for nonobvious reasons".

**Comment by SpaceManiac** [comment]
Turns out this worked before and was broken accidentally in 990e61050c616ff61d1fdf7475c6d5f499bc8aab.

```
error: got ':', expected one of: '/', '[', contents, newline, identifier
```

Fixed as of e7ce6f6136c76bd32110fcca9882a02775ada156.

```
warning: path separated by ':', should be '/'
```
